pipelines/testing/CompareVcfs.wdl

```diff
@@ -0,0 +1,39 @@
+version 1.0
+
+workflow CompareVcfs {
+
+    input {
+        File test_gvcf
+        File truth_gvcf
+    }
+
+    call CompareVcfs {
+        input:
+            file1 = test_gvcf,
+            file2 = truth_gvcf,
+            patternForLinesToExcludeFromComparison = "^##"
+    }
+}
+
+task CompareVcfs {
+    input {
+        File file1
+        File file2
+        String patternForLinesToExcludeFromComparison
+    }
+    Int disk_size_gb = ceil(3 * size(file1, "GiB")) + ceil(3 * size(file2, "GiB")) + 50
+    command {
+        set -eo pipefail
+
+        gunzip -c -f ~{file1} | grep -v '~{patternForLinesToExcludeFromComparison}' > file_1.vcf
+        gunzip -c -f ~{file2} | grep -v '~{patternForLinesToExcludeFromComparison}' > file_2.vcf
+
+        diff -q file_1.vcf file_2.vcf
```

Comment:
is there a way to capture some output to illustrate what the diff is if it fails? it'd be nice to know what the differences are, but currently the log just says 
```Files file_1.vcf and file_2.vcf differ```

Reply:
yea we probably wouldnt want to use diff for that cuz the output would be super hard to read.  We'd want another tool to run when we're expecting differences.  This would be mostly meant to make sure things are the same for changes we expect to not make a difference.  (i can remove the `-q` to get the actual differences but it was pretty unreadable to me)

Reply:
gotcha, makes sense